### ISSUE TITLE
Add cache invalidation info to remove warnings

### ISSUE THREAD

**Issue by GavinJoyce** [ember-animation/liquid-fire#485]
I'm seeing these warnings when booting with ember-cli 2.7:

```
DEPRECATION: ember-cli-htmlbars-inline-precompile is opting out of caching due to an AST plugin that does not provide a caching strategy: `transform-liquid-with-as-to-hash`.
```

Context:

 * https://embercommunity.slack.com/archives/-ember-cli/p1470864929001722
 * https://github.com/babel/broccoli-babel-transpiler/pull/89

**Comment by GavinJoyce** [comment]
@rwjblue is this another case of adding `baseDir` to:

 * https://github.com/ember-animation/liquid-fire-core/blob/a74852e31a1b6af54704c54e60ae23b8152a4b1a/index.js#L46-L52
 * https://github.com/ember-animation/liquid-fire-velocity/blob/a74852e31a1b6af54704c54e60ae23b8152a4b1a/index.js#L46-L53

(I'm not sure if both of those are used or now)

**Comment by rwjblue** [comment]
Yep, they need the same treatment.

**Comment by GavinJoyce** [comment]
Thanks, I'll cut some PRs now

**Comment by GavinJoyce** [comment]
I've created two PRs:

 * https://github.com/ember-animation/liquid-fire-core/pull/2
 * https://github.com/ember-animation/liquid-fire-velocity/pull/2

I'm not sure how `ext/plugins` gets included with `liquid-fire`. I can see it in my `node_modules`, but it doesn't seem to be in the main `liquid-fire` repo.

**Comment by rwjblue** [comment]
Looks like it was removed in https://github.com/ember-animation/liquid-fire/commit/37c52099557af3241d5c8af670aff4c87cc7909e. I believe that is included in 0.24.1 so you may be able to update and not have the deprecation?

**Comment by GavinJoyce** [comment]
@rwjblue ah, thanks. I checked the released page and saw `0.23.0` and thought that we were running the latest. Not the greatest UI I think:

<img width="796" alt="screen shot 2016-08-12 at 17 59 35" src="https://cloud.githubusercontent.com/assets/2526/17630323/bb1c2dba-60b6-11e6-81f7-81f9e7bcd2ef.png">

Thanks
